github/pkg/adapter/adapter.go

```diff
@@ -19,79 +19,213 @@ package adapter
 import (
 	"context"
 	"fmt"
-	"log"
 	"net/http"
 	"strconv"
 	"strings"
+	"time"
 
-	cloudevents "github.com/cloudevents/sdk-go"
+	sourcesv1alpha1 "knative.dev/eventing-contrib/github/pkg/apis/sources/v1alpha1"
```

Comment:
Let's group this import together with the other third-party packages. Or even better: group all `knative.dev` imports together.

Reply:
Thank for your review. I wil think about them.  :)

Reply:
Done.